src/browser_action.js

```diff
@@ -96,11 +97,12 @@ browseraction.loadCalendarsIntoQuickAdd_ = function() {
   chrome.extension.getBackgroundPage().background.log('browseraction.loadCalendarsIntoQuickAdd_()');
   chrome.storage.local.get('calendars', function(storage) {
     if (chrome.runtime.lastError) {
-      background.log('Error retrieving calendars:', chrome.runtime.lastError);
+      chrome.extension.getBackgroundPage().background.log(
+          'Error retrieving calendars:', chrome.runtime.lastError);
     }
 
-    if (storage['calendars']) {
-      var calendars = storage['calendars'];
+    if (storage.calendars) {
```

Comment:
This is a JS property, but on line 98, it’s a string constant, which is likely to get out of sync. Any thoughts? (The obvious one to me is to use a defined string constant at both places.)

Reply:
Done.

Reply:
...incorrectly. I'll follow up momentarily.

Reply:
Done.